### PR TITLE
CAL-291 copy edits to sources and applications docs (#631) 

### PR DESCRIPTION
**Pull request by ricklarsen**
**Port to `master` of PR https://github.com/codice/alliance/pull/631**
#### What does this PR do?

Copy edits to documentation application reference sections.

#### Who is reviewing it? 
@mcalcote @ahoffer @austinsteffes @garrettfreibott 

#### Choose 2 committers to review/merge the PR.

@clockard
@lessarderic
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested?

content changes only.

#### What are the relevant tickets?

[CAL-291](https://codice.atlassian.net/browse/CAL-291)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_reference/_applicationReferences/mg-video.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ None.
 
 === Installing ${alliance-video}
 
-The ${alliance-video} application is not installed by default on a standard installation. To install:
+To install the ${alliance-video} application:
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_reference/_applicationReferences/mg-video.adoc]: fixed

**Comment by cxbot1** [comment]
Internal build has been scheduled, your results will be available at build completion.

**Comment by cxbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.codice.org/job/Alliance-Jobs/job/pr/job/Linux/1010/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='Alliance-Jobs/pr/Linux/org.codice.alliance:docs' /><a href='https://jenkins.codice.org/job/Alliance-Jobs/job/pr/job/Linux/1010/org.codice.alliance$docs/testReport'>Alliance-Jobs/pr/Linux/org.codice.alliance:docs</a>: <span class='status-failure'>1</span></h3><ul><li><a href='https://jenkins.codice.org/job/Alliance-Jobs/job/pr/job/Linux/1010/org.codice.alliance$docs/testReport/alliance.docs/DocumentationTest/testBrokenAnchorsPresent/'><strong>alliance.docs.DocumentationTest.testBrokenAnchorsPresent</strong></a></li></ul>
:x: JOB FAILURE

**Comment by ricklarsen** [comment]
previous build failed due to upstream artifacts not deployed. Will get manual verification...

**Comment by austinsteffes** [comment]
manually verified on 4422a40 👍

**Pull request merged**